server/service/src/requisition/request_requisition/insert_program.rs

```diff
@@ -158,6 +165,13 @@ fn validate(
     ))
 }
 
+pub(super) struct GenerateResult {
```

Comment:
noticed that response and request were using the same GenerateResult, updated for them to be separate